utils/context_manger.py

```
@@ -0,0 +1,40 @@
+
+    def search_db(self)-> bool:
+        pass
+
+    def get_cache(self)-> ContextResponse:
+        pass
+
+    def get_db(self)-> ContextResponse:
+        pass
+
+    def get_context(self)-> ContextResponse:
+        if self.check_redis(self.query):
```

Pylint (3.3.7)

[error] Instance of 'FetchContext' has no 'check_redis' member

(E1101)